.gitignore

```diff
@@ -3,4 +3,6 @@
 
 build/
 .gradle/
-.jmix
\ No newline at end of file
+.jmix
+jmix-studio.xml
+jmix-studio.xml
```

Comment:
What have changed?) Still two `jmix-studio.xml` entries and it still has conflict with the **Master** branch